nixos/modules/programs/sway-beta.nix

```diff
@@ -5,6 +5,15 @@ with lib;
 let
   cfg = config.programs.sway-beta;
   swayPackage = cfg.package;
+
+  swayWrapped = pkgs.writeShellScriptBin "sway" ''
+    ${cfg.extraSessionCommands}
+    exec ${pkgs.dbus.dbus-launch} --exit-with-session ${swayPackage}/bin/sway
```

Comment:
Needed to make dbus dependend programs to work (like libnotify notifications). Also official wiki recommends launching via dbus-launch for one particular case https://github.com/swaywm/sway/wiki#im-not-using-logind-but-still-want-dbuspolkitpower-management-to-work

Reply:
I just packaged [`mako`](https://github.com/emersion/mako) in [my overlay][1] an hour ago and just noticed it wasn't working.

This patch fixes it. Thanks.

[1]: https://github.com/colemickens/nixpkgs-wayland/commit/5428f63e83c2e9ae98f2d90ae9d193b447c6ca7a

Reply:
@colemickens In some cases you would also need `exec dbus-update-activation-environment DISPLAY` in your sway config